lib/modules/datasource/glasskube-packages/index.ts

```diff
@@ -0,0 +1,75 @@
+import { logger } from '../../../logger';
+import { joinUrlParts } from '../../../util/url';
+import { Datasource } from '../datasource';
+import type { GetReleasesConfig, ReleaseResult } from '../types';
+import {
+  GlasskubePackageManifestYaml,
+  GlasskubePackageVersions,
+  GlasskubePackageVersionsYaml,
+} from './schema';
+
+export class GlasskubePackagesDatasource extends Datasource {
+  static readonly id = 'glasskube-packages';
+  static readonly defaultRegistryUrl =
+    'https://packages.dl.glasskube.dev/packages';
+
+  override defaultRegistryUrls = [
+    GlasskubePackagesDatasource.defaultRegistryUrl,
+  ];
+
+  constructor() {
+    super(GlasskubePackagesDatasource.id);
+  }
+
+  override async getReleases({
```

Comment:
```suggestion
  @cache({
    namespace: `datasource-${GlasskubePackagesDatasource.id}`,
    key: ({ packageName }: GetReleasesConfig) => `${packageName}`,
  })
  override async getReleases({
```
Let's cache the results